src/main.ts

```diff
@@ -43,29 +44,52 @@ async function main() {
   await solRouter.refresh_dips();
   solRouter.run_risk_manager();
   if (!IS_DEV) {
+    await sleepExact(staggerTime);
     await btcRouter.refresh_dips();
     btcRouter.run_risk_manager();
+    await sleepExact(staggerTime);
     await ethRouter.refresh_dips();
     ethRouter.run_risk_manager();
   }
 
   setInterval(async () => {
-      console.log('Rerun All Risk Managers', new Date().toUTCString());
+      console.log('Rerun SOL Risk Manager', new Date().toUTCString());
       try {
         await solRouter.refresh_dips();
         solRouter.run_risk_manager();
-        if (!IS_DEV) {
-          await btcRouter.refresh_dips();
-          btcRouter.run_risk_manager();
-          await ethRouter.refresh_dips();
-          ethRouter.run_risk_manager();
-        }
       } catch (err) {
         console.log(err);
         console.log(err.stack);
       }
     }, 1_000 * (scalperWindow + (((Math.random()*2) - 1) * scalperWindow * percentDrift))
   );
+
+  if (!IS_DEV) {
+    await sleepExact(staggerTime);
+    setInterval(async () => {
+      console.log('Rerun BTC Risk Manager', new Date().toUTCString());
+      try {
+        await btcRouter.refresh_dips();
+        btcRouter.run_risk_manager();
+      } catch (err) {
+        console.log(err);
+        console.log(err.stack);
+      }
+    }, 1_000 * (scalperWindow + (((Math.random()*2) - 1) * scalperWindow * percentDrift))
```

Comment:
You still want them all on the same interval so they dont drift, just want them smeared. Should be able to do it with sleeps right after run_risk_manager for each

Reply:
Will this trim down the last products total run time though? ie. Will ETH gamma scalper only be in reality running for 9 mins

Reply:
setInterval only sets when to start, not how long to run for. It allows overlapping calls